File: src/main/java/aero/sort/vizualizer/data/options/styles/concrete/Aqua.java
// -------------------------------------------------------------------------------
// Copyright (c) Ladendorfer Daniel.  
// All Rights Reserved.  See LICENSE in the project root for license information.
// -------------------------------------------------------------------------------
package aero.sort.vizualizer.data.options.styles.concrete;

import aero.sort.vizualizer.data.options.styles.AbstractGradientStyle;
import aero.sort.vizualizer.ui.constants.Theme;

import java.awt.*;

/**
 * Aqua gradient style.
 *
 * @author Daniel Ladendorfer
 */
public class Aqua extends AbstractGradientStyle {
    @Override
    protected Color getPrimaryColor() {
        return Theme.BLUE;
    }

    @Override
    protected Color getSecondaryColor() {
        return Theme.DEEPBLUE;
    }
}

File: src/main/java/aero/sort/vizualizer/data/options/styles/concrete/gradient/Aqua.kt
// -------------------------------------------------------------------------------
// Copyright (c) Ladendorfer Daniel.  
// All Rights Reserved.  See LICENSE in the project root for license information.
// -------------------------------------------------------------------------------
package aero.sort.vizualizer.data.options.styles.concrete.gradient

import aero.sort.vizualizer.ui.constants.Theme
import java.awt.Color

/**
 * Aqua gradient style.
 *
 * @author Daniel Ladendorfer
 */
class Aqua : AbstractGradientStyle() {
    override fun getPrimaryColor(): Color = Theme.BLUE
    override fun getSecondaryColor(): Color = Theme.DEEP_BLUE
}